README.md

```diff
@@ -12,14 +12,34 @@
 </p><br>
 
 ## Installation
-You can install q2-fondue using mamba in a conda environment of its own (option 1) or in an existing QIIME 2 environment (option 2) by following the steps described below. The current q2-fondue version supports QIIME 2 **v2021.4** or higher. To install q2-fondue with a version <= 2023.7 see section ["Installing q2-fondue with version <= 2023.7"](#legacy). Alternatively, a minimal Docker image is available to run q2-fondue v2023.7 (option 3 below - note other Docker image q2-fondue versions are work in progress).
+There are multiple options to install q2-fondue - each targeted towards different needs. You can install q2-fondue (**v2024.5** or higher):
+* as part of the QIIME 2 metagenome distribution if you want to use it in the QIIME 2 metagenome universe (see [option 1](#option-1-install-q2-fondue-with-qiime-2-metagenome-distribution)).
+* within a QIIME 2 amplicon environment if you want to use it in the QIIME 2 amplicon universe (see [option 2](#option-2-install-q2-fondue-within-a-qiime-2-amplicon-conda-environment)).
+* in a conda environment of its own if you want to use it independent of other QIIME 2 tools (see [option 3](#option-3-minimal-fondue-environment)).
 
-For both options 1 and 2 make sure to start by installing [mamba](https://mamba.readthedocs.io/en/latest/index.html) in your base environment:
+To install q2-fondue with a version <= 2023.7 see [this](#installing-q2-fondue-with-older-versions) section.
+
+### Option 1: Install q2-fondue with QIIME 2 metagenome distribution
+* Install q2-fondue together with the QIIME 2 metagenome distribution as outlined [here](https://docs.qiime2.org/2024.5/install/native/#qiime-2-metagenome-distribution).
```

Comment:
do not harcode to a specific version, this will go stale. I suggest just referring users to the QIIME 2 installation instructions.

It is also vaguely written here, as it is not clear that q2-fondue is installed as part of the QIIME 2 metagenome distribution. You should just say so directly.